acceptance/.beaker.yml

```diff
@@ -0,0 +1,15 @@
+---
+ssh:
+  keys:
+    - id_rsa_acceptance
+    - ~/.ssh/id_rsa-acceptance
+xml:                         true
+timesync:                    false
+repo_proxy:                  true
+add_el_extras:               false
+forge_host:                  forge-aio01-petest.puppetlabs.com
+'master-start-curl-retries': 30
+log_level:                   debug
+preserve_hosts:              onfail
+helper:                      ./lib/helper.rb
+options_file:                ./config/aio/options.rb
```

Comment:
I already added a comment on this, but it was on the commit, and it got lost somewhere, so I'm re-adding it here

Could you all a bit more information to the commit message as to why you are defaulting to aio? Also, it would be nice to include a TODO comment, since we intend to change this once we fix git acceptance